.github/workflows/android-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Build & publish viamrtsp module for Android
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-android:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.21
+    
+    - name: Build viamrtsp binary and package module
+      run: |
+        TARGET_OS=android TARGET_ARCH=arm64 make module
+    
+    - name: Upload viamrtsp module to registry
+      uses: viamrobotics/upload-module@main
+      if: startsWith(github.ref, 'refs/tags/') 
```

Comment:
Why is this line needed?

Reply:
This checks for a tag event. (Had used it in previous modules).

The example wifi module uses `if: github.event_name == 'release'` (see[ here](https://github.com/viam-labs/wifi-sensor/blob/main/.github/workflows/deploy.yml#L29C7-L30C1)) -- but even this seems redundant with the on publish check.

Reply:
Ok, removed if check since we do not run it in publish.yml and should not be necessary.